rtc_data_service/src/attestation/service.rs

```diff
@@ -0,0 +1,89 @@
+use std::convert::TryInto;
+
+use actix::{Addr, MailboxError};
+use actix_web::{error::ErrorInternalServerError, post, web, HttpRequest};
+use models::*;
+use rtc_types::{AttestError, AttestationResponse};
+use rtc_uenclave::AttestationError;
+
+use crate::data_enclave_actor::DataEnclaveActor;
+use crate::merge_error::*;
+
+use super::AttestationMessage;
+
+#[post("auth/attest")]
+pub async fn req_attestation_jwt(
+    _req: HttpRequest,
+    req_body: web::Json<RequestBody>,
+    enclave: web::Data<Addr<DataEnclaveActor>>,
+) -> actix_web::Result<web::Json<ResponseBody>> {
+    let message: AttestationMessage = req_body.0.try_into()?;
+
+    let result: Result<AttestationResponse, MergedError<AttestationError, MailboxError>> =
+        enclave.send(message).await.merge_err();
+    match result {
+        Ok(resp) => Ok(web::Json(resp.into())),
+        Err(err) => Err(ErrorInternalServerError(err)),
+    }
+}
+
+pub mod models {
```

Comment:
Since we don't have a request body, or response struct (at this point), we can just remove the custom models here.